docassemble/Covid19debt/data/questions/DebtReport.yml

```diff
@@ -431,22 +422,21 @@ fields:
     choices: 
       - Renting: Renting
         help: | 
-          You rent if you pay someone who owns the home to live in your house or apartment
```

Comment:
@nonprofittechy It is still removing my additions. I want these added in where "Information" is.